group_text/group_text.php

```diff
@@ -56,7 +58,7 @@ function group_text_settings_post($a,$post) {
 
 
 
-function group_text_settings(&$a,&$s) {
+function group_text_settings(App $a, $s) {
```

Comment:
The `&` operator of the `$s` parameter enables this function to change the hook data. If you remove it this function has no effect whatsoever.